### PR TITLE
feat: 1039 update seedlot related tests to use stored variable

### LINKED ISSUE #1039: Update seedlot related tests to use stored variable
From our pair programming sessions we have defined a way to store seedlot registration data object with the species as the key.

This issue should implement the following:
- [x] `create-a-class-seedlot.cy.ts` Register the rest of 4 seedlots using the `PLI` example
- [x] `seedlot-dashboard.cy.ts` Verify the 5 seedlots generated by importing the fixture, instead of checking with hardcoded string values, e.g. `cy.should('have.text', 'FDC - Coastal Douglas-fir');` should become `.should('have.text', 'fixtureData.fdc.sepcies');`. Do **not** count the total number of rows, but rather check the 5 species of seedlot that were registered earlier. Get the species text and status with id using the seedlot number stored, as you can see in the screenshot the seedlot number is within each id. 
Image

- [x] `edit-applicant-seedlot-info.cy.ts` Get rid of the hardcoded seedlot number 63001, use the dynamically stored value by calling `cy.task('getData', YOUR_KEY).then((value) => { })`; After you changed the fields, verify that they are indeed changed on the seedlot detail page, then **change it back**. This is so that we can still use the fixture file as our reference later on.

### PR DESCRIPTION
**Pull request by ngunner15**
# Description
Closes #1039 

#### New
-

#### Changed
- Made the following tests more dynamic by removing hardcoded values
- seedlot-dashboard
- edit-applicant-seedlot-info

#### Removed
-

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [x] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-34-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1084-backend.apps.silver.devops.gov.bc.ca/actuator/health)
- [Oracle-API](https://nr-spar-1084-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

**Review by RMCampos — APPROVED** [review]
LGTM! Great job 💯

File: frontend/cypress/e2e/smoke-test/seedlot-dashboard.cy.ts
Comment (craigyu) [inline frontend/cypress/e2e/smoke-test/seedlot-dashboard.cy.ts:1]: image

This test isn't following the specification. Use seedlot number in the element id to get the species and status text. Do not make assumption that the seedlot in a specific row will be under a species. If there are existing seedlots or the test ran multiple times, how can we know for sure that the first row is what we just registered?

**Review by craigyu — CHANGES_REQUESTED** [review]

**Review by craigyu — APPROVED** [review]
Amazing loop

**Comment by ngunner15** [comment]
> Amazing loop

Finally able to loop some tests :)

**Pull request merged**